test/studies/stencil9/stencil9-explicit.chpl

```diff
@@ -19,24 +20,26 @@ const LocDom = {1..n  , 1..n  },
 const LocaleGridDom = Dom._value.dist.targetLocDom,
       LocaleGrid = Dom._value.dist.targetLocales;
```

Comment:
I think you can now use "Dom.targetLocales()", and ".domain" on the result to avoid the hacky _value stuff.